index.html

```diff
@@ -1,140 +1,170 @@
+				developers who want to contribute by writing code for Git for Windows.</p>
+			<p>Please look at the <a href="https://github.com/git-for-windows/git/wiki/Technical-overview">technical
+					overview</a> of the Git for Windows packaging and how to include your changes in your own custom
+				installer.</p>
+			<a name="download-sdk" /><a class="btn btn-outline-primary shadow mr-2"
+				href="https://github.com/git-for-windows/build-extra/releases/latest" target="_blank">Download <span
+					class="gittext">Git for Windows SDK</span></a>
+			<p>To contribute/update this web page, see its <a
+					href=" https://github.com/git-for-windows/git-for-windows.github.io">Repository</a>.</p>
+		</div>
+	</section>
+
+	</div>
+	</div>
+
+	<footer id="dk-footer" class="dk-footer">
+		<div class="copyright">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-6">
+						<span>Copyright © 2020 Johannes Schindelin and contributors. All rights reserved.</span>
 					</div>
-					<div class="vcentercontainer detailstext">
-						<div>
-							<p>As an open source project, Git for Windows benefits greatly from both the volunteer work of helpful developers and <a href="https://github.com/git-for-windows/git/wiki/Issue-reporting-guidelines">good bug reports</a> made by Git for Windows' users.</p>
-							<h3>Governance Model</h3>
-							<p>If you are interested in contributing to the <i>Git for Windows</i> project, have a look at the current <a href="governance-model.html">governance model</a>. There the roles of those involved in driving this project further down the road are described in detail.</p>
-							<h3><a name="issues"></a>Bug Reports &amp; Feature Requests</h3>
-							<p>If you've noticed a bug or simply have an idea that you'd like to see become real, why not work on it?  Bug reports and feature requests are typically submitted to the issue tracker on our GitHub repository.</p>
-							<a class="button" href="https://github.com/git-for-windows/git/issues" target="_blank">Issue Tracker</a>
-							<h3><span class="gittext">Git for Windows SDK</span> : Contributing Code</h3>
-							<p>The <i>Git for Windows SDK</i> is a build environment that includes all the tools necessary for developers who want to contribute by writing code for Git for Windows.</p>
-							<p>Please look at the <a href="https://github.com/git-for-windows/git/wiki/Technical-overview">technical overview</a> of the Git for Windows packaging and how to include your changes in your own custom installer.</p>
-							<a name="download-sdk" /><a class="button" href="https://github.com/git-for-windows/build-extra/releases/latest" target="_blank">Download <span class="gittext">Git for Windows SDK</span></a>
-							<p>To contribute/update this web page, see its <a href=" https://github.com/git-for-windows/git-for-windows.github.io">Repository</a>.</p>
+					<!-- End Col -->
+					<div class="col-md-6">
+						<div class="copyright-menu">
```

Comment:
Does this really need to be called "copyright-menu"? There is nothing about any copyright in that section...